UnitTests/PaymentsCoreTests/EligibilityParsing_Tests.swift

```diff
@@ -0,0 +1,86 @@
+import XCTest
+@testable import PaymentsCore
+@testable import Card
+
+class EligibilityParsing_Tests: XCTestCase {
+
+    func testFundingEligibility_matchesResponse() throws {
+        if let jsonData = validFundingEligibilityResponse.data(using: .utf8) {
+            let fundingEligibilityObject = try JSONDecoder().decode(FundingEligibilityResponse.self, from: jsonData)
+            XCTAssertTrue(fundingEligibilityObject.fundingEligibility.venmo.eligible == true)
+            XCTAssertTrue(fundingEligibilityObject.fundingEligibility.card.eligible == false)
```

Comment:
Can we add assertions for the other payment methods as well? `paypal`, `credit`, `paylater`